### PR TITLE
DMT-30 Add labels to the partitions of the visualization

### PR DESCRIPTION
**Pull request by eemilj**
## Related issue
- Helps DMT-30

## Proposed changes
- Adds corresponding percentage value labels to each partition in the donut

## Additional information 
- Not a final implementation, corner cases to be added, font styling needs to be fixed relative to visualization size etc.

### CODE REVIEW

File: src/renderer.js [hunk 1]
```diff
@@ -43,14 +46,57 @@ export async function render(donutState) {
         })
         .attr("fill", () => "transparent");
 
+
     sectors
         .merge(newSectors)
+        .attr("class", "sector")
         .transition()
         .duration(animationDuration)
+        .attr("value", (d) => (d.data.value / donutState.sumOfValues * 100).toFixed(1))
```
Comment (krasen86) [inline src/renderer.js:55]: Can we extract/encapsulate this into a function, so we avoid code repetition as in 76 and 91? Or maybe even have it as an attribute to the data?

File: src/renderer.js [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 /**
- * @param {donutState} donutState
+ * @param {object} donutState
  */
 export async function render(donutState) {
     // Added a constant to remove the magic numbers within the width, height and radius calculations.
     const sizeModifier = 40;
     // D3 animation duration used for svg shapes
-    const animationDuration = 300;
+    const animationDuration = 200;
```
Comment (clementinejensen) [inline src/renderer.js:9]: Do we know what animation speed is preffered by Tibco? Is there a set standard?

**Review by krasen86 — COMMENTED** [review]
Looks good

File: src/renderer.js (hunk 1 above)
Comment (eemilj) [reply src/renderer.js]: Yes!

**Review by clementinejensen — APPROVED** [review]
Looks great!

**Pull request merged**